packages/@o3r/components/src/tools/placeholder/placeholder.component.ts

```diff
@@ -52,22 +53,27 @@ export class PlaceholderComponent implements OnInit, OnDestroy {
     this.subscription.add(
       this.id$.pipe(
         distinctUntilChanged(),
-        switchMap((id) => this.store.select(selectPlaceholderRenderedTemplates(id)))
-      ).subscribe((templates) => {
-        if (templates) {
-          this.isPending = templates.isPending;
-          const orderedRenderedTemplates = templates.orderedRenderedTemplates;
-          if (!orderedRenderedTemplates || !orderedRenderedTemplates.length) {
-            this.template = undefined;
-          } else {
-            // Concatenates the list of templates
-            this.template = orderedRenderedTemplates.join('');
-          }
-        } else {
-          this.isPending = false;
+        switchMap((id) =>
+          this.store.select(selectSortedRenderedPlaceholder(id)).pipe(
+            map((placeholders) => ({id, orderedRenderedPlaceholders: placeholders?.orderedRenderedPlaceholders, isPending: placeholders?.isPending}))
```

Comment:
```suggestion
            map((placeholders) => ({id, ...placeholders}))
```
?

Reply:
If I do that, then the properties from placeholder might not exist in the subscription callback and I am not sure that would be more readable:
``subscription(data => {
const isPending = data.isPending;
const orderedTemplates = data.orderedTemplates
}``